lib/vsc/utils/run.py

```diff
@@ -86,6 +86,48 @@
 SHELL = BASH
 
 
+class CmdList(list):
+    """Wrapper for 'list' type to be used for constructing a list of options & arguments for a command."""
+
+    def __init__(self, cmd=None):
+        """
+        Create CmdList instance to construct command
+
+        :param cmd: actual command to run (first item in list)
+        """
+        super(CmdList, self).__init__()
+        if cmd is not None:
```

Comment:
initial command is not sanity checked? can contain spaces, can be a non string thing?

Reply:
Good point, I'll fix that by using `add_opts_args` here too.